tajo-core/src/main/java/org/apache/tajo/master/QueryInProgress.java

```diff
@@ -53,9 +56,9 @@
 
   private LogicalRootNode plan;
 
-  private AtomicBoolean querySubmitted = new AtomicBoolean(false);
+  private volatile boolean querySubmitted = false;
 
-  private AtomicBoolean stopped = new AtomicBoolean(false);
+  private AtomicBoolean isStopped = new AtomicBoolean(false);
```

Comment:
This variable also seems to be able to be defined as a volatile boolean instead of AtomicBoolean.